# 05 EIA report day

Is the absolute RWTC move larger on Wednesdays than on other published days, in both windows?

Holdout starts 2020-01-01. Verdict: held.

This study reads stored series. It does not add a derived column.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'report-day')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train ratio of mean absolute Wednesday log return to other days is 1.059. Holdout ratio is 1.043. Both windows clear 1, which was the bar. The margin is small.
Mean absolute adjacent RWTC log return
Train | Wednesday | 1584 | 1.852%
Train | Other published days | 6183 | 1.750%
Holdout | Wednesday | 348 | 2.209%
Holdout | Other published days | 1336 | 2.118%


## Method

Compare the mean absolute log return on those Wednesdays with the mean absolute log return on every other published day.

## Result

Train ratio of mean absolute Wednesday log return to other days is 1.059. Holdout ratio is 1.043. Both windows clear 1, which was the bar. The margin is small.